client/js/libs/handlebars/firstLetterUppercase.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+module.exports = function(string) {
+	return string.charAt(0).toUpperCase() + string.slice(1);
+};
```

Comment:
This is no longer in this file, it's in the server. So it's a bit more complicated to test just now. I could pull it out into a module but that feels overkill fo rsomething so small just now.